Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:20-alpine3.17
+FROM node:20-alpine3.18
 
 RUN apk add --no-cache python3 make g++  \
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)